### PR TITLE
Implement SearchSuggestions component

### PR DESCRIPTION
**Pull request by reynard2007**
PR for Task #7 to replace PR #32

### CODE REVIEW

**Comment by vikasrohit** [comment]
Merging. Final fixes would be done later on.

File: components/SearchSuggestions/SearchSuggestions.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+'use strict'
```
Comment (nlitwin) [inline components/SearchSuggestions/SearchSuggestions.jsx]: Can remove 'use strict'

File: components/SearchSuggestions/SearchSuggestions.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+require('./SearchSuggestions.scss')
+const React = require('react')
```
Comment (nlitwin) [inline components/SearchSuggestions/SearchSuggestions.jsx]: `import React from 'react'`

File: components/SearchSuggestions/SearchSuggestions.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import StandardListItem from '../StandardListItem/StandardListItem.jsx'
+import Panel from '../Panel/Panel'
+
+const SearchSuggestions = {
```
Comment (nlitwin) [inline components/SearchSuggestions/SearchSuggestions.jsx]: convert to ES6 class and put getInitialState in constructor (see comments on SearchBar (https://github.com/appirio-tech/react-components/pull/35/files)

File: components/SearchSuggestions/SearchSuggestionsExamples.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+'use strict'
```
Comment (nlitwin) [inline components/SearchSuggestions/SearchSuggestionsExamples.jsx]: Can remove

File: components/SearchSuggestions/SearchSuggestionsExamples.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+const React = require('react')
```
Comment (nlitwin) [inline components/SearchSuggestions/SearchSuggestionsExamples.jsx]: `imports` > `requires`

**Pull request merged**